src/dvc_render/vega_templates.py

```diff
@@ -441,10 +435,8 @@ class ScatterTemplate(Template):
                 "type": "quantitative",
                 "title": Template.anchor("y_label"),
             },
-            "color": {
-                "field": "rev",
-                "type": "nominal",
-            },
+            "color": Template.anchor("color"),
+            "shape": Template.anchor("shape"),
```

Comment:
[F] Using different stroke dash for scatter templates results in some less-than-ideal results:

<img width="1200" alt="image" src="https://github.com/iterative/dvc-render/assets/37993418/701840e8-fa5f-4112-97d9-0ce724a9f22d">